src/AppBundle/Controller/Admin/StatisticsController.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace AppBundle\Controller\Admin;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @Route("admin/statistics")
+ */
+class StatisticsController extends Controller
+{
+    private $months = [
+        "01" => "Januari",
+        "02" => "Februari",
+        "03" => "Mars",
+        "04" => "April",
+        "05" => "Maj",
+        "06" => "Juni",
+        "07" => "Juli",
+        "08" => "Augusti",
+        "09" => "September",
+        "10" => "Oktober",
+        "11" => "November",
+        "12" => "December"
+    ];
+
+    /**
+     * @Route("/", name="admin_statistics")
+     */
+    public function indexAction(Request $request)
+    {
+        $city = $request->query->get("city", 2);
+        $year = $request->query->get("year", date("Y"));
+        $type = $request->query->get("type", "");
+
+        $matches = $this->getConnectionRepository()->getMatches($city, $year, $type);
+        $matches = $this->structuredMatches($matches);
+
+        $cities = $this->getCityRepository()->findAll();
+
+        $allConnections = $this->getConnectionRepository()->getAllConnections();
+        $years = $this->getYearSpan($allConnections);
+
+        $parameters = [
+            "matches" => $matches,
+            "cities" => $cities,
+            "years" => $years,
+            "params" => array(
+                "city" => $city,
+                "year" => $year,
+                "type" => $type,
+            )
+        ];
+
+        return $this->render('admin/statistics/index.html.twig', $parameters);
+    }
+
+    protected function getYearSpan($connections)
+    {
+        $years = [];
+        foreach ($connections as $connection) {
+            $year = date("Y", strtotime($connection["created_at"]));
+            if (!in_array($year, $years)) {
+                $years[] = $year;
+            }
+        }
+        return $years;
+    }
+
+    protected function structuredMatches($matches)
```

Comment:
I'm not sure I know a better solution for this.. Do you got any best practice on how to populate the year selection?